sound/hybrid_font.h

```diff
@@ -120,7 +132,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (swing_player_) {
+        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+        // between 0 (no overlap) and 1.0 (full overlap)
+        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
+          RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+          swing_player_ = PlayPolyphonic(&swng);
+          overlap_swing->set_fade_time(overlap_swing->length() - overlap_swing->pos());
+          overlap_swing->FadeAndStop();
+          overlap_swing.Free();
+        }
+      }
+      else if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+        swing_player_->set_volume_now(0);
```

Comment:
This is weird, we should not need to do this.
If this is what makes the crackle go away, then I have a theory:
When a new swing starts, it starts at default volume, which may be significantly higher than what we will set it to in SetSwingVolume(), and maybe the rapid changing of the volume is what causes the crackling.

We should probably be doing two things:
1) set the volume to the correct value, as calculated by SetSwingVolume() *before* we start playing the swing.
2) I'm not sure what the default fade time is, but we probably want to set it to something like 20ms or something as that avoids introducing a lot of noise from setting the volume too rapidly.

Reply:
Ok I’ll test that out.

Reply:
All I had to do was set fade_time to 2 just before setting volume in SetSwingVolume and all odd pops and crackles went away.